Packages/manifest.json

```diff
@@ -1,10 +1,12 @@
 {
+  "registry": "https://staging-packages.unity.com",
   "dependencies": {
     "com.unity.ads": "2.0.8",
-    "com.unity.analytics": "2.0.16",
-    "com.unity.package-manager-ui": "1.9.11",
+    "com.unity.analytics": "3.0.9",
+    "com.unity.collab-proxy": "1.2.9",
+    "com.unity.package-manager-ui": "2.0.0-preview.7",
     "com.unity.purchasing": "2.0.3",
-    "com.unity.textmeshpro": "1.2.4",
+    "com.unity.textmeshpro": "1.3.0",
```

Comment:
Please revert changes to this file.

Reply:
on it.

Reply:
reverted to 492289b7d33dba5269844c73c2c958112463bf07